src/App.css

```diff
@@ -10,7 +10,6 @@
     justify-content: space-between;
     font-size: 1.25;
     color: white;
-    max-height: 6vh;
```

Comment:
point of this?

Reply:
it was lookin real weird

Reply:
if we're setting the height of the page we can leave this relative